cmd/server/api/universalmintingrpchandler.go

```diff
@@ -167,7 +176,7 @@ func tokenByIndex(callData erc721.CallData, params ParamsRPCRequest, stateServic
 	sendResponse(w, tokenId.String(), err)
 }
 
-func createMerkleTrees(tx state.Tx, contactAddress common.Address) (state.Tx, error) {
+func loadMerkleTree(tx state.Tx, contactAddress common.Address, blockNumber string) (state.Tx, error) {
```

Comment:
Rename `contactAddress` to `contractAddress` please

Reply:
changed